src/js/tinymce_plugins/schematags.js

```diff
@@ -283,8 +282,8 @@
 				} else {
 					isLevel1 = false;
 				}
-				
-				if (att.name.toLowerCase() != 'id' && att.name.toLowerCase() != 'xml:id') {
+
+				if (att.name.toLowerCase() != 'xid' && att.name.toLowerCase() != 'xml:xid') {
```

Comment:
Instead of changing to something which will not match (so this condition is always `true`), should this condition not just be eliminated entirely? As in, this line remove (along with the closing brace).